play/src/front/Connection/RoomConnection.ts

```diff
@@ -231,6 +232,14 @@ export class RoomConnection implements RoomConnection {
     public readonly askMutedMessage = this._askMutedMessage.asObservable();
     private readonly _askMutedVideoMessage = new Subject<AskMutedVideoMessage>();
     public readonly askMutedVideoMessage = this._askMutedVideoMessage.asObservable();
+    private readonly _proximityPrivateMessageToClientMessageStream =
+        new Subject<ProximityPrivateMessageToClientMessage>();
+    public readonly proximityPrivateMessageToClientMessageStream =
+        this._proximityPrivateMessageToClientMessageStream.asObservable();
+    private readonly _proximityPublicMessageToClientMessageStream =
+        new Subject<ProximityPublicMessageToClientMessage>();
+    public readonly proximityPublicMessageToClientMessageStream =
+        this._proximityPublicMessageToClientMessageStream.asObservable();
```

Comment:
New subjects declared (`_proximityPublicMessageToClientMessageStream` and `_proximityPrivateMessageToClientMessageStream`) must be properly completed in the `completeStreams()` method.
Otherwise, we have a memory leak when moving rooms.